src/App.vue

```diff
@@ -214,7 +218,9 @@ export default {
 
     var self = this
 
-    this.socket = io(ConfigApis.getSocketIP())
+    this.socket = this.socket = io('/', {
```

Comment:
not sure what the double `=` is doing here, mistake or does it have some functionality I don't know about?
```suggestion
    this.socket = io('/', {
```

Reply:
Nope -- syntactically correct, but a complete oversight!